.circleci/config.yml

```diff
@@ -850,6 +850,12 @@ jobs:
       - run_test:
           pattern: 'pytest$'
 
+  asynctest:
```

Comment:
we're not actually running this scenario

we have to add it to the list of jobs to run: eg. https://github.com/DataDog/dd-trace-py/blob/bbe53881f2a718d46ef943401b1ca30a45cf8747/.circleci/config.yml#L1211